routes/apply_discount.js

```diff
@@ -232,6 +234,55 @@ const responseSchema = {
   'type': 'object',
   'additionalProperties': false,
   'properties': {
+    'buy_together': {
+      'type': 'array',
+      'maxItems': 20,
+      'items': {
+        'type': 'object',
+        'additionalProperties': false,
+        'maxProperties': 50,
+        'patternProperties': {
+          '^([a-f0-9]{24})$': {
+            'type': 'object',
+            'required': [ 'quantity' ],
```

Comment:
```suggestion
            'required': [ 'discount' ],
```

Como aqui é resposta do app de descontos, acho que faz sentido requerer o `discount`, certo?
Se não tiver desconto nem deveria retornar o `buy_together` e tal...
O `quantity` acho que pode ser opcional mesmo.